mantidproject/mantid
# Add <hollow-cylinder> as recognised shape in XML definitions

The `ShapeFactory` has been extended to understand the concept of a `hollow-cylinder`. See the referenced issue for more detail.

**To test:**

Review the code and check the example in the unit tests makes sense.

As a test for visualization I modified the MARI instrument definition locally to use a hollow cylinder instead of a cylinder for the fermi-chopper. The instrument view then shows the expected shape:

![hollowcylinderinsrtumentview](https://cloud.githubusercontent.com/assets/733773/15178503/df3a9534-176d-11e6-9fd0-db5c83084ce5.png)

Try something similar to convince yourself it works as expected.

Fixes #16132

[Release notes updated](/mantidproject/mantid/commit/c7770f9c3b534949b08ce01c9129bbab32fc9425)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
